__tests__/app.test.js

```diff
@@ -420,4 +420,69 @@ describe('/api/comments/:comment_id', () => {
         })
     });
 
+});
+
+describe('/api/articles/:article_id/comments', () => {
+    describe('GET comments by article_id', () => {
+
+        test.only('/api/articles/:article_id/comments responds 200 with array of comments to article  ', () => {
+            return request(app).get('/api/articles/3/comments').expect(200).then((response) => {
+
+
+                expect(response.body.comments).toBeInstanceOf(Array);
+
+                expect(response.body.comments.length).toBe(2);
+
+                response.body.comments.forEach((comment) => {
+                    expect(comment).toEqual(expect.objectContaining({
+
+                        comment_id: expect.any(Number),
+                        votes: expect.any(Number),
+                        body: expect.any(String),
+                        created_at: expect.any(String),
+                        author: expect.any(String)
+                    }))
+                })
+
+            })
+
+        })
+
+        test.only('/api/articles/article:id/comments responds error 404 when wrong path been passed ', () => {
+            return request(app).get('/api/articles/3/comment').expect(404).then((response) => {
+
+                const message = { msg: "Path not found" };
+                expect(response.body).toEqual(message);
+            })
+
+        })
+
+
+        test.only('/api/articles/article:id/comments responds error 404 when article_id does not exist in DB', () => {
+            return request(app).get('/api/articles/1234/comments').expect(404).then((response) => {
+
+                const message = { msg: "Resource not found" };
+                expect(response.body).toEqual(message);
+            })
+
+        })
+        test.only('/api/articles/article:id/comments responds error 404 when article has not been commented', () => {
+            return request(app).get('/api/articles/2/comments').expect(404).then((response) => {
```

Comment:
This one should really be a 200 - as the article exists and there are simply no associated comments 👍